jena-core/src/test/java/org/apache/jena/test/TestPackage.java

```diff
@@ -32,7 +33,7 @@
 public class TestPackage extends TestCase {
 
```

Comment:
`Test_rdfcat` now has a lot of deprecation warnings. Is it just a case of `@SuppressWarnings("deprecation")` on the class or is there something else going on?